modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/friendly/url/test/JournalArticleFriendlyURLTest.java

```diff
@@ -134,6 +135,27 @@ public void testFriendlyURLAfterUpdate() throws Exception {
 		Assert.assertEquals(friendlyURLMap, updatedArticle.getFriendlyURLMap());
 	}
 
+	@Test
+	public void testFriendlyURLShouldNotHaveSiteLanguageNotEqualDefaultLanguage()
+		throws Exception {
+
+		LocaleThreadLocal.setSiteDefaultLocale(LocaleUtil.US);
```

Comment:
Hi @AliciaGarciaGarcia, all looks great! Just the comment that we talked about. Would it make sense set the current value again after the test finishes in the thread local?

Reply:
Thanks, I will send an additional commit.